<launch>
    <!-- 加载参数文件 -->
    <rosparam file="$(find naruto_closedloop)/config/params.yaml" command="load" />

    <!-- 启动节点 -->
    <node name="position_control" pkg="naruto_closedloop" type="position_control" output="screen" />
    <node name="serial_port" pkg="laser_uart" type="serial_port" output="screen" />
        
</launch>